### PR TITLE
feat(docs): migration guide for `LocalFiles` and `outputDir`

### PR DESCRIPTION
**Pull request by wrussell1999**
Migration Guide to help users change their flows using `LocalFiles` and `outputDir`

### CODE REVIEW

**Comment by wrussell1999** [comment]
@anna-geller wasn't sure what the best name for this file was as it's all around making handling input and output files for tasks better. Let me know what you think of `/task-files`.

**Comment by anna-geller** [comment]
What about Input Files? It's all about providing additional files as inputs to the task

File: content/docs/11.migration-guide/task-files.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
+---
+title: Depreciation of LocalFiles and outputDir
```
Comment (anna-geller) [inline content/docs/11.migration-guide/task-files.md:2]:
```suggestion
title: Deprecation of LocalFiles and outputDir
```

**Comment by wrussell1999** [comment]
> Usually, we name migration guide after the thing that is deprecated so it seems best to call the file `local-files.md` - WDYT?

Ah yeah that's a good shout, updated

File: content/docs/11.migration-guide/task-files.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+icon: /docs/icons/migration-guide.svg
+release: 0.17.0
+---
+
+`LocalFiles` and `outputDir` were depreciated in Kestra 0.17.0. Here is why:
```
Comment (anna-geller) [inline content/docs/11.migration-guide/task-files.md]:
```suggestion
Check out this guide to migrate from `LocalFiles` and `outputDir` to `inputFiles` and `outputFiles`.

## Overview
The `LocalFiles` and `outputDir` are deprecated due to overlapping functionality that already exists using `inputFiles` and `outputFiles` on the `WorkingDirectory` and [script](https://kestra.io/docs/developer-guide/scripts) tasks.
```

keep in mind the first sentence must be short to be displayed on the card preview + social card

**Review by anna-geller — APPROVED** [review]
thanks a lot!

File: content/docs/11.migration-guide/input-files.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
+---
+title: Deprecation of LocalFiles and outputDir
+icon: /docs/icons/migration-guide.svg
+release: 0.17.0
+---
+
+`LocalFiles` and `outputDir` were depreciated in Kestra 0.17.0. Here is why:
```
Comment (anna-geller) [inline content/docs/11.migration-guide/input-files.md:7]:
```suggestion
Check out this guide to migrate from `LocalFiles` and `outputDir` to `inputFiles` and `outputFiles`.

## Overview
The `LocalFiles` and `outputDir` are deprecated due to overlapping functionality that already exists using `inputFiles` and `outputFiles` on the `WorkingDirectory` and [script](https://kestra.io/docs/developer-guide/scripts) tasks.
```

keep in mind the first sentence must be short to be displayed on the card preview + social card

**Comment by wrussell1999** [comment]
Renamed to `local-files` and description added. Sorry for the confusion there!

**Pull request merged**